packages/experimental/Expander/windows/ReactNativeExpander/ExpanderViewManager.cpp

```diff
@@ -0,0 +1,133 @@
+#include "pch.h"
```

Comment:
I'm a little confused as to the nature of this change overall... I thought the idea was to use react-native-xaml to expose the control to JS, but it seems like this change is just hand-rolling duplicated logic that we already have in place?